bin/anycablebility

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+
+require 'bundler/setup'
+require 'docopt'
```

Comment:
"Active Years Age" takes into an account issues and PRs. Yep, kind of confusing.

For the other hand, the [commits history](https://github.com/docopt/docopt.rb/commits/master) shows the real situation. As well as the state of open issues and PRs.